Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -850,7 +848,6 @@ private void CoinstakeWorker(CoinstakeWorkerContext context, ChainedBlock chainT
                             context.Result.KernelCoin = coin;
 
                             context.Logger.LogTrace("Kernel accepted, coinstake input is '{0}/{1}', stopping work.", prevoutStake.Hash, prevoutStake.N);
-                            kernelFound = true;
```

Comment:
You rely on this to break out of the loop?
` if (context.Result.KernelFoundIndex != CoinstakeWorkerResult.KernelNotFound)`

Reply:
` if (stopWork) break;` this one